### ISSUE TITLE
UglifyJS Build Error

### ISSUE THREAD

**Issue by bbar** [developit/mitt#51]
Hey guys, I opened an [issue](https://github.com/react-boilerplate/react-boilerplate/issues/1705) with [React Boilerplate](https://github.com/react-boilerplate/react-boilerplate), but thought I'd open one here as well to see if you've ever come across problems uglifying this module installed via npm. Maybe there's a problem with the Webpack config and I just need to add something to make it play nice? Anyway, the boilerplate is one of the most popular boilerplates available for React and it yields a build error when Mitt is either a direct dependency, or any dependent packages are using Mitt.

Here are the steps to reproduce:

```
$ git clone --depth=1 https://github.com/react-boilerplate/react-boilerplate.git
$ cd react-boilerplate
$ npm run setup
$ npm run clean
$ yarn add mitt
```

Open react-boilerplate/app/containers/HomePage/index.js and add the following import:

`import mitt from 'mitt';`

Then,

`$ npm run build`

The given will be something like this:

```
ERROR in 1.673e1d9e06dc9452619e.chunk.js from UglifyJs
SyntaxError: Unexpected token punc «(», expected punc «:» [1.673e1d9e06dc9452619e.chunk.js:105,4]
```

**Comment by developit** [comment]
This happens when Uglify encounters ES2015 code. @lukeed has PR in that fixes it: #48.

**Comment by bbar** [comment]
Rad, you guys already know about it. Thanks for the quick response.
